lib/SauceLabs.js

```diff
@@ -268,6 +270,9 @@ function generateLink(hostname, auth, date, job) {
 }
 
 function makeRequest(options, body, callback) {
+    if(process.env.https_proxy !=='undefined' && process.env.https_proxy) {
```

Comment:
Yes I think so; I don't mind the module supporting both but someone shouldn't have to change how they use the module :)